plugin.rb

```diff
@@ -200,7 +177,7 @@ def enabled?
 end
 
 auth_provider title_setting: "oauth2_button_title",
-              authenticator: OAuth2BasicAuthenticator.new('oauth2_basic'),
+              authenticator: OAuth2BasicAuthenticator.new(),
```

Comment:
Nitpick: the brackets are not required if there are no arguments. Ditto in a few other places

Reply:
Fair. I've removed these :+1: